infer/resource.go

```diff
@@ -926,10 +951,12 @@ func (rc *derivedResourceController[R, I, O]) Create(ctx p.Context, req p.Create
 	return p.CreateResponse{
 		ID:         id,
 		Properties: m,
-	}, nil
+	}, err
```

Comment:
How does returning an explicit value interact with the deferred editing of the named returns? Which one wins?

Reply:
By returning an error, it is assigned to `retError`. The `defer`ed function get's to manipulate `retError` (and `resp`) to manipulate the value ultimately returned by `Create`.